cluster-provision/base/scripts/dnsmasq.sh

```diff
@@ -9,6 +9,12 @@ ip link add br0 type bridge
 ip link set dev br0 up
 ip addr add dev br0 192.168.66.02/24
 
+# Create secondary networks
```

Comment:
Could you run DHCP servers on each network? It would help us with testing on kubernetes-nmstate.

Reply:
It will not, in fact we are flushing them at cluster-up https://github.com/nmstate/kubernetes-nmstate/blob/master/hack/flush-secondary-nics.sh

Reply:
What do you mean? We flush the assigned IPs initially to get a clean start. Then however, during test run, we create policies obtaining IP on those ifaces via DHCP. So we need a DHCP server.

Reply:
It may fit a separate PR, but I doubt we will find a good reason to have it as a service of the infra.
We could probably stick with static addresses and if a need to test with dynamic ones appears, a dedicated pod can be ran with the DHCP server.

Reply:
Ahh right, let me investigate if we can make dnsmasq run on different bridges...

Reply:
It *might* be needed to have each dnsmasq on a different netns - not really sure, please try without first.

Reply:
IMO we should not have this in the infra. A dhcp server should be loaded by the tests in a pod connected to that network.

Reply:
The provider gives its user a prepared infrastructure with hosts connected through networks. When I'm given an infrastructure, I would expect that DHCP is part of the package. We can of course do via a pod on the client-side, but I see it as an unnecessary effort for knmstate and other users.

Reply:
@phoracek is ok we do this at follow up PR ? it's only needed for kubernetes-nmstate.

Reply:
@phoracek IMO using dynamic IP is a bad idea in an enterprise env, and these secondary networks are supposed to represent such enterprise env if I understand correctly.
I think it will be better to control if you want DHCP or not.

But if you still think it is better to have it by default and without the options to control it from the tests, then a following PR can propose it indeed.

Reply:
Make sense. I don't mind dnsmasq being called explicitly by kubevirtci user (e.g. knmstate).